plugins/fortinet_fortigate/icon_fortinet_fortigate/actions/get_address_objects/action.py

```diff
@@ -38,13 +38,13 @@ def run(self, params={}):
 
         params = {"filter": filters}
         endpoint = "firewall/address"
-        results = self.connection.api.get_address_objects(endpoint, params).get("results")
+        results = self.connection.api.get_address_objects(endpoint, params).get("results", [])
 
-        for i, result in enumerate(results):
```

Comment:
@dsliwinski-r7 please revert this - I don't think I saw in the diff previously that you were using the `i` variable here which is why I suggested the `_` to ignore that. Using `i` is fine - sorry about that!!